Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Tests/Get-ExchangeEmergencyMitigationServiceState.Tests.ps1

```diff
@@ -0,0 +1,192 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingInvokeExpression', '', Justification = 'Pester testing file')]
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidOverwritingBuiltInCmdlets', '', Justification = 'Pester testing file')]
+[CmdletBinding()]
+param()
+BeforeAll {
+    $Script:parentPath = (Split-Path -Parent $PSScriptRoot)
+    $Script:Server = $env:COMPUTERNAME
+    . $Script:parentPath\Get-ExchangeEmergencyMitigationServiceState.ps1
+
+    # We need to overwrite the existing 'Get-Service' function because it doesn't support
+    # the '-ComputerName' parameter on PowerShell 7
+    Function Get-Service {
+        param(
+            [Parameter(Mandatory = $false)]
+            [string]
+            $ComputerName,
+            [Parameter(Mandatory = $false)]
+            [string]
+            $Name
+        )
+    }
+}
+
+Describe "Testing Get-ExchangeEmergencyMitigationServiceState.ps1" {
+
+    Context "Exchange Emergency Mitigation Service Default State" {
+
+        BeforeAll {
+            Mock Invoke-WebRequest -MockWith { return Import-Clixml $Script:parentPath\Tests\InvokeWebRequestEEMS.xml }
+            Mock Get-Service -MockWith { return Import-Clixml $Script:parentPath\Tests\GetServiceMSExchangeMitigationRunning.xml } `
+                -ParameterFilter { $ComputerName -eq $Script:Server }
+
+            $customObject = ([PSCustomObject]@{
+                    ComputerName       = $Script:Server
```

Comment:
Recommend changing this to a function that you can call and pass the parameters that are changing in your `Context`

Reply:
I've just changed the properties within the `BeforeAll` under `Context`. Looks like it do the job well.